### DISCUSSION TITLE
Show total leaderboard length

### DISCUSSION THREAD

**Discussion by memesnightmares** [ppy/osu#19157] [Ideas]
When I'm not in the top 50, it'd be cool to see how many people I'm ahead of or behind. Perhaps just change the number to show <my_rank>/<leaderboard_length>.

**Comment by frenzibyte** [comment]
Related: https://github.com/ppy/osu/discussions/17673

**Comment by memesnightmares** [comment]
I believe this is completed now, with SSv2
